e2e/test/scenarios/question/reproductions/38454-changing-source-database.cy.spec.js

```diff
@@ -0,0 +1,44 @@
+import {
+  getNotebookStep,
+  openNotebook,
+  popover,
+  restore,
+  visualize,
+} from "e2e/support/helpers";
+import { ORDERS_ID } from "metabase-types/api/mocks/presets";
+
+const QUESTION_DETAILS = {
+  query: {
+    "source-table": ORDERS_ID,
+    limit: 5,
+  },
+};
+
+describe("issue 38354", () => {
```

Comment:
```suggestion
describe("issue 38354", { tags: "@external" }, () => {
```

The same comment as here, but I will not block the PR.
https://github.com/metabase/metabase/pull/38387#discussion_r1476031587

Feel free to push the change with `[ci skip]` and I'll force merge it.

Reply:
964b5787fa854d44ca3df0040d3741fecd2aa9eb